pkg/virtctl/vm/restart.go

```diff
@@ -65,16 +65,14 @@ func (o *Command) restartRun(args []string, cmd *cobra.Command) error {
 		return fmt.Errorf("Must both use --force=true and set --grace-period.")
 	}
 
+	restartOpts := &v1.RestartOptions{DryRun: dryRunOption}
 	if forceRestart {
-		err = virtClient.VirtualMachine(namespace).ForceRestart(context.Background(), vmiName, &v1.RestartOptions{GracePeriodSeconds: &gracePeriod, DryRun: dryRunOption})
-		if err != nil {
-			return fmt.Errorf("Error force restarting VirtualMachine, %v", err)
-		}
-	} else {
-		err = virtClient.VirtualMachine(namespace).Restart(context.Background(), vmiName, &v1.RestartOptions{DryRun: dryRunOption})
-		if err != nil {
-			return fmt.Errorf("Error restarting VirtualMachine %v", err)
-		}
+		restartOpts.GracePeriodSeconds = &gracePeriod
```

Comment:
`gracePeriod` seems to default to `-1`, is that even a valid value? Shouldn't it default to `0`?

Reply:
Actually, this is reached only if gracePeriod is manually set. Otherwise the restartRun function would fail earlier:
```
	if gracePeriodChanged != forceRestart {
		return fmt.Errorf("Must both use --force=true and set --grace-period.")
	}
```
Thank you!
I changed the reported error, since it was causing an e2e failure.